Makefile

```diff
@@ -120,8 +120,8 @@ mockery:
 
 #? check-proto-deps: Check protobuf deps
 check-proto-deps:
-ifeq (,$(shell which protoc-gen-gogofaster))
-	@go install github.com/cosmos/gogoproto/protoc-gen-gogofaster@latest
+ifeq (,$(shell which protoc-gen-gocosmos))
```

Comment:
`gogofaster` is outdated. `gocosmos` is used by [latest](https://github.com/cometbft/cometbft/blob/d2d5577205dd9fa30ce43430964543065cfea7d3/Makefile#L123)